tree/30_generic_methods/file_line_present_in_ini_section.cf

```diff
@@ -34,32 +34,60 @@ bundle agent file_line_present_in_ini_section(file, section, line)
       "report_param"           string => join("_", args);
       "full_class_prefix"      string => canonify("file_line_present_in_ini_section_${report_param}");
       "class_prefix"           string => string_head("${full_class_prefix}", "1000");
-      "section_and_blank_line" string => "[${section}]
-";
+      "hash"                   string => hash("${report_param}", "md5");
```

Comment:
this seems overkill. file and section should be enough

Reply:
you can have 2 edit in the same sections and file in 2 directives, this is fine